CSharpBasics/CSharpOOP2/Horse.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpOOP2
+{
+    public class Horse
+    {
+        public string NickName;
+        public string Color;
+        public double Age;
+        public string Breed;
+        public double TailLength;
+        public bool IsVaccinated;
+        public bool IsHealthy;
+        public int DaysNumberLastVaccination;
+        public int NumbersOfWins = 5;
+        public int _speed;
```

Comment:
Public fields and properties should start with capitalized letter :) only private ones start with _